superset-frontend/src/components/ButtonGroup/ButtonGroup.stories.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import Button from 'src/components/Button';
+import ButtonGroup from './index';
+
+export default {
```

Comment:
Maybe a minor thing, but I kind of prefer named exports to default exports. I'm not sure where this pattern is going to settle in Superset, but I like that named imports are typically easier to search for in the codebase.

Reply:
@rusackas In this specific case I'm using default export following Storybook [documentation](https://storybook.js.org/docs/react/api/csf): 

> The default export defines metadata about your component, including the component itself, its title (where it will show up in the navigation UI story hierarchy), decorators, and parameters.

Now looking at the rest of the application, I prefer named exports also but currently the majority of our components are using default exports and the screens using them are importing using component's folder name. Let me describe here how I'm going to tackle this:

- First I'm going to remove the difference between common components and components. All will be components with their own folder, storybook and test. For this step, I will preserve current standard and use default exports.
- Remove Bootstrap dependency
- After all components are following the same structure, we will create an `index` in components folder exporting our version of the components using named exports. That way we'll import our version using `import { Component } from 'src/components';`
- Changing the code to use named exports will be done in a specific PR.